easylist/easylist_adservers.txt

```diff
@@ -9333,6 +9349,7 @@
 ||gam3ah.com^
 ||gambar123.com^
 ||gameads.io^
+||gamefocus.mobi^
```

Comment:
@smed79 Not looks like an adserver, at least not a revolving server?

Reply:
I doubt it is legitimate site, it has something to to with revolving server. Probably redirection or fake ads.

It's just a generic template, look at the contact info at the these pages:
```
gamefocus.mobi/contact-us/
qsmh.org/contact-us/
primedesign.co.za/contact-us/
wirraldeencentre.org.uk/contact/
sciencebug.in/contact-us/
wavepro.nl/contact-us/
pathwaysempowerment.com/contact-us/
```

Anyway, I deleted the site for the benefit of the doubt.